pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>io.burt</groupId>
   <artifactId>athena-jdbc</artifactId>
-  <version>0.2.0</version>
+  <version>0.3.0-SNAPSHOT</version>
```

Comment:
Perhaps you need to modify this to make sure the changes work, but in general, only change project versions on master or in release-only branches. The reason is that the current version may be 0.4 or 9.1 by the time this gets merged.

Reply:
Good point, reverted.